integration_tests/testsuite/test_util.py

```diff
@@ -50,13 +63,18 @@ def _generate_hex_token():
     return binascii.b2a_hex(os.urandom(16))
 
 
+def _generate_log_level():
+    return random.choice(list(Severity)).name
+
+
 def _generate_int64_token():
     return random.randint(-(2 ** 31), (2 ** 31)-1)
 
 
 def generate_logging_payload():
```

Comment:
What do you think about running the test for each log level, instead of just picking one?